README.md

```diff
@@ -59,9 +68,31 @@ If you are using open-source config-generator UI, you need to set the parameter
 $ helm install my-release ./ --set backend.controlPlaneJSON=true
  ```
 
+## Extending the Chart
+
+Since we are publishing the Chart under the {{ TBC by the RudderStack team }} page. It's possible to extend this Chart 
+by adding it as a dependency into your own Chart, so there is no need to git clone this repo for deploying RudderStack
+open-source into your infrastructure.
+
+```yaml
+apiVersion: v2
+name: rudderstack
+description: Customer Data Pipeline tool for collecting, routing and processing data.
+maintainers:
+  - name: Data Platform
+    email: xxxx@xxxx.com
+version: 0.4.5
+appVersion: 1.16.0
+dependencies:
+  # https://github.com/rudderlabs/rudderstack-helm
+  - name: rudderstack
+    version: 0.4.5
+    repository: https://TBC.github.io/rudderstack-helm # To Be Confirmed by the RudderStack team
```

Comment:
@lokey , once the RudderStack configures the gh-pages to host the Chart index, could you provide or update this section with the right URL?